rust_src/src/keymap.rs

```diff
@@ -5,13 +5,17 @@ use remacs_sys::{current_global_map as _current_global_map, globals, EmacsInt, L
                  CHAR_META};
 use remacs_sys::{Fcons, Fevent_convert_list, Ffset, Fmake_char_table, Fpurecopy, Fset};
 use remacs_sys::{Qautoload, Qkeymap, Qkeymapp, Qnil, Qt};
-use remacs_sys::{access_keymap, maybe_quit};
+use remacs_sys::{access_keymap, map_keymap_call, map_keymap_function_t, map_keymap_internal,
+                 maybe_quit};
 
 use data::{aref, indirect_function};
 use eval::autoload_do_load;
 use keyboard::lucid_event_type_list_p;
+use libc::c_void;
```

Comment:
`libc` is an external crate. It belongs at the top, before remacs_sys.

Reply:
I see. Got it.